app/services/meeting_service.rb

```diff
@@ -0,0 +1,215 @@
+module MeetingService
```

Comment:
Use bang methods and add some extra rescues if you want to return something like true/false